disnake/__main__.py

```diff
@@ -22,29 +22,42 @@ def show_version():
     entries.append(
         f"- disnake v{disnake_ver.major}.{disnake_ver.minor}.{disnake_ver.micro}-{disnake_ver.releaselevel}"
     )
-    if pkg := pkg_resources.get_distribution("disnake"):
-        entries.append(f"    - disnake pkg_resources: v{pkg.version}")
+    try:
+        version = importlib.metadata.version("disnake")
+    except importlib.metadata.PackageNotFoundError:
+        pass
+    else:
+        entries.append(f"    - disnake importlib.metadata: v{version}")
 
     entries.append(f"- aiohttp v{aiohttp.__version__}")
     uname = platform.uname()
     entries.append(f"- system info: {uname.system} {uname.release} {uname.version}")
```

Comment:
```suggestion
    entries.append(f"- system info: {uname.system} {uname.release} {uname.version} {uname.machine}")
```
Figured it would be useful to include the machine type. This results in the following outputs for me:
- linux, 64 bit: `Linux 5.19.0-76051900-generic #202207312230~1660780566~22.04~9d60db1 SMP PREEMPT_DYNAMIC Thu A x86_64`
- windows: 64 bit: `Windows 7 6.1.7601 AMD64`
- linux, armv7: `Linux 5.15.61-v7l+ #1579 SMP Fri Aug 26 11:13:03 BST 2022 armv7l`